src/python/bot/fuzzers/syzkaller/stats.py

```diff
@@ -0,0 +1,332 @@
+# Copyright 2019 Google LLC
```

Comment:
This file need to be removed, should be added for syzkaller case when we know which stats it exports.

Reply:
Done.